<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="vLayout_Center">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widgetControl" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QHBoxLayout" name="hLayoutControle">
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="tButtonMenu">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="icons.qrc">
           <normaloff>:/icons/menu.png</normaloff>:/icons/menu.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
         <property name="popupMode">
          <enum>QToolButton::InstantPopup</enum>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBarUpdate">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>3</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"> QProgressBar {
   background-color: palette(base);
      border: 0px solid grey;
      border-radius: 1px;
  }

  QProgressBar::chunk {
      background-color: palette(highlight);
     border-radius: 1px;
   }</string>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetMessage" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
    background-color:palette(highlight);
    color:palette(highlight-text)
}
QToolButton {
   border-radius:5;
   border:1px solid transparent;
	background-color:  palette(highlight);
}

QToolButton:hover {
   color: palette(window-text);
   background-color:  palette(shadow);
}
</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>There are folders that can be updated</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tButtonCancelMsg">
         <property name="text">
          <string>Cancel</string>
         </property>
         <property name="autoRaise">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tButtonOkMsg">
         <property name="text">
          <string>Update</string>
         </property>
         <property name="autoRaise">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageLibrery">
       <layout class="QVBoxLayout" name="vLayoutLibrery">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="handleWidth">
           <number>5</number>
          </property>
          <property name="childrenCollapsible">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="widgetContent" native="true">
           <layout class="QVBoxLayout" name="vLayoutContent">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QComboBox" name="comboBoxSwichCat">
              <item>
               <property name="text">
                <string>Genre / Artst</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Genre / Album</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Album / artist</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Artiste / Album</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Genre / Artiste / Album</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Genre / Album / Artiste </string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Arist</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Album</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widgetView" native="true">
           <layout class="QVBoxLayout" name="vLayoutViewAll">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>2</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widgetStatus" native="true">
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_path">
               <property name="spacing">
                <number>1</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>1</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>1</number>
               </property>
               <item>
                <widget class="QToolButton" name="tb_paneContent">
                 <property name="toolTip">
                  <string>Show/Hide Contents</string>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="icons.qrc">
                   <normaloff>:/icons/pane-hide.png</normaloff>:/icons/pane-hide.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="shortcut">
                  <string notr="true">Ctrl+K</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelStatusG">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelStatusP">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelStatus">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QToolButton" name="tb_panePlaylist">
                 <property name="toolTip">
                  <string>Show/Hide Playlist</string>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                 <property name="icon">
                  <iconset resource="icons.qrc">
                   <normaloff>:/icons/pane-show.png</normaloff>:/icons/pane-show.png</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>16</width>
                   <height>16</height>
                  </size>
                 </property>
                 <property name="shortcut">
                  <string notr="true">Ctrl+L</string>
                 </property>
                 <property name="checkable">
                  <bool>true</bool>
                 </property>
                 <property name="autoRaise">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QStackedWidget" name="stackedWidgetView">
              <property name="styleSheet">
               <string notr="true">QTreeView{
                   background-color: transparent;
                   border-color: palette(Window);
color:palette(window-text);
                   }
QListView{
                   background-color: transparent;
                   border-color: palette(Window);
                  color:palette(window-text);
                   }</string>
              </property>
              <property name="currentIndex">
               <number>1</number>
              </property>
              <widget class="QWidget" name="page_View">
               <layout class="QVBoxLayout" name="vLayoutView">
                <property name="spacing">
                 <number>3</number>
                </property>
                <property name="leftMargin">
                 <number>1</number>
                </property>
                <property name="topMargin">
                 <number>3</number>
                </property>
                <property name="rightMargin">
                 <number>3</number>
                </property>
                <property name="bottomMargin">
                 <number>3</number>
                </property>
               </layout>
              </widget>
              <widget class="QWidget" name="pageAudio">
               <layout class="QHBoxLayout" name="hlLayoutAudios">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>3</number>
                </property>
                <property name="topMargin">
                 <number>3</number>
                </property>
                <property name="rightMargin">
                 <number>3</number>
                </property>
                <property name="bottomMargin">
                 <number>3</number>
                </property>
                <item>
                 <widget class="QWidget" name="widgetCurAlbum" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>130</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>130</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_2">
                   <property name="leftMargin">
                    <number>1</number>
                   </property>
                   <property name="topMargin">
                    <number>1</number>
                   </property>
                   <property name="rightMargin">
                    <number>1</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QLabel" name="labelAlbumImage">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                     <property name="pixmap">
                      <pixmap resource="icons.qrc">:/icons/cover.svg</pixmap>
                     </property>
                     <property name="scaledContents">
                      <bool>false</bool>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignBottom|Qt::AlignHCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_2">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QToolButton" name="tb_addAlbum">
                       <property name="toolTip">
                        <string>Add curent Album</string>
                       </property>
                       <property name="icon">
                        <iconset resource="icons.qrc">
                         <normaloff>:/icons/album-add.png</normaloff>:/icons/album-add.png</iconset>
                       </property>
                       <property name="autoRaise">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QToolButton" name="tb_favoritAlbum">
                       <property name="toolTip">
                        <string>Add album to favorit</string>
                       </property>
                       <property name="icon">
                        <iconset resource="icons.qrc">
                         <normaloff>:/icons/star.png</normaloff>:/icons/star.png</iconset>
                       </property>
                       <property name="checkable">
                        <bool>true</bool>
                       </property>
                       <property name="autoRaise">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QToolButton" name="tb_imgAlbum">
                       <property name="toolTip">
                        <string>change cover image</string>
                       </property>
                       <property name="icon">
                        <iconset resource="icons.qrc">
                         <normaloff>:/icons/preview.png</normaloff>:/icons/preview.png</iconset>
                       </property>
                       <property name="autoRepeat">
                        <bool>true</bool>
                       </property>
                       <property name="autoExclusive">
                        <bool>true</bool>
                       </property>
                       <property name="autoRaise">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QToolButton" name="tb_playAlbum">
                       <property name="toolTip">
                        <string>play curent album</string>
                       </property>
                       <property name="styleSheet">
                        <string notr="true"/>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset resource="icons.qrc">
                         <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
                       </property>
                       <property name="autoRaise">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                   <item>
                    <spacer name="verticalSpacer">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>17</width>
                       <height>134</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
